core/templates/pages/exploration-player-page/layout-directives/exploration-footer.component.html

```diff
@@ -42,11 +42,11 @@
           </div>
         </li>
       </ul>
-      <oppia-hint-and-solution-buttons *ngIf="windowIsNarrow"></oppia-hint-and-solution-buttons>
+      <oppia-hint-and-solution-buttons *ngIf="windowIsNarrow && !resultsLoaded"></oppia-hint-and-solution-buttons>
```

Comment:
The footer is also used in question-player-page. Hints should only be shown on the footer when user is playing through the questions. However after user has finished the questions the score is loaded on the same page and now no hints should be shown. So we set results loaded to true which removes hints-solution from footer once user has finished practising.

Reply:
If `resultsLoaded` do not have any meaning in the exploration context, we should probably use a better name so that it makes sense in the question as well as exploration contexts e.g. `hintsAndSolutionsAreSupported`.

Reply:
Done